apitest/src/test/java/bisq/apitest/method/RegisterDisputeAgentsTest.java

```diff
@@ -42,10 +44,14 @@
 @TestMethodOrder(OrderAnnotation.class)
 public class RegisterDisputeAgentsTest extends MethodTest {
 
+    private static final String ARBITRATOR = "arbitrator";
```

Comment:
I guess those constants would be good to have in the domain classes and maybe enums are even better.